src/style/markdown.less

```diff
@@ -1,9 +1,16 @@
 .markup, .markdown {
-	& > *:first-child {
-		margin-top: 0;		// !important
+	& > * ~ * {
```

Comment:
Also known as the [lobotomized owl selector](https://alistapart.com/article/axiomatic-css-and-lobotomized-owls/)